### PR TITLE
feat: more lru cache metrics

### PR DESCRIPTION
**Pull request by hanabi1224**
**Summary of changes**
Changes introduced in this pull request:
- Add prometheus metrics for `tipset_cache` and `skip_cache`

Sample output
```
# HELP lru_cache_hit Stats of lru cache hit
# TYPE lru_cache_hit counter
lru_cache_hit{kind="skip"} 42258
lru_cache_hit{kind="tipset"} 20602
# HELP lru_cache_miss Stats of lru cache miss
# TYPE lru_cache_miss counter
lru_cache_miss{kind="skip"} 1983
lru_cache_miss{kind="tipset"} 40236
```

**Reference issue to close (if applicable)**

Closes https://github.com/ChainSafe/forest/issues/2188

**Other information and links**

### CODE REVIEW

File: blockchain/chain/src/store/index.rs [hunk 1]
```diff
@@ -75,8 +75,14 @@ impl<BS: Blockstore> ChainIndex<BS> {
         loop {
             let entry = self.skip_cache.write().await.get(&cur).cloned();
             let lbe = if let Some(cached) = entry {
+                metrics::LRU_CACHE_TOTAL
+                    .with_label_values(&[metrics::values::SKIP_LRU_HIT])
```
[the remark this answers is not in the document]
Comment (hanabi1224) [reply blockchain/chain/src/store/index.rs:79]: Fixed

File: blockchain/chain/src/store/metrics.rs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+pub mod labels {
+    pub const LRU_CACHE_KIND: &str = "lru_cache_kind";
+}
+
+pub mod values {
+    // lru_cache_total
+    pub const TIPSET_LRU_HIT: &str = "tipset_lru_hit";
+    pub const TIPSET_LRU_MISS: &str = "tipset_lru_miss";
+    pub const SKIP_LRU_HIT: &str = "skip_lru_hit";
+    pub const SKIP_LRU_MISS: &str = "skip_lru_miss";
```
[the remark this answers is not in the document]
Comment (hanabi1224) [reply blockchain/chain/src/store/metrics.rs]: Fixed

**Comment by LesnyRumcajs** [comment]
```
# HELP lru_cache_hit Stats of lru cache hit
# TYPE lru_cache_hit counter
lru_cache_hit{lru_cache_hit_kind="skip_lru_hit"} 1171
lru_cache_hit{lru_cache_hit_kind="tipset_lru_hit"} 1751
# HELP lru_cache_miss Stats of lru cache miss
# TYPE lru_cache_miss counter
lru_cache_miss{lru_cache_miss_kind="skip_lru_miss"} 273
lru_cache_miss{lru_cache_miss_kind="tipset_lru_miss"} 4661
```
it's a bit redundant, what do you think about having only `skip` and `tipset` labels? The key as well. So something along the lines of:
```
lru_cache_hit{lru_cache_kind="skip"} 1171
```

**Comment by hanabi1224** [comment]
> what do you think about having only skip and tipset labels? 

@LesnyRumcajs  Fixed. That's much better, thanks!

**Review by LesnyRumcajs — APPROVED** [review]
LGTM, thanks!

**Review by lemmih — APPROVED** [review]

**Pull request merged**